__tests__/server/utils/circuitBreaker.spec.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import CircuitBreaker from 'opossum';
+import breaker, {
+  setEventLoopLagThreshold,
+  getEventLoopLagThreshold,
+} from '../../../src/server/utils/circuitBreaker';
+
+describe('Circuit breaker', () => {
+  it('should be an opossum circuit breaker', () => {
```

Comment:
does this matter if everything else works ?

Reply:
not necessarily